bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -8,164 +8,311 @@
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
     {% endblock %}
 
-      <p>
-        {{ _('Check out "What’s New" and "Known Issues" for this version of Firefox below.') }}
-      {% trans feedback='https://input.mozilla.org/feedback/android/' + release.version + '?utm_source=releasenotes'
-          if release.product == 'Firefox for Android'
-          else 'https://input.mozilla.org/feedback/firefox/' + release.version + '?utm_source=releasenotes',
-          bugzilla='https://bugzilla.mozilla.org/' %}
-        As always, you’re encouraged to
-        <a href="{{ feedback }}">tell us what you think</a>,
-        or <a href="{{ bugzilla }}">file a bug in Bugzilla</a>.
-      {% endtrans %}
-      {% if release.bug_list %}
-      <form action="https://bugzilla.mozilla.org/buglist.cgi" method="post">
-        <input type="hidden" name="limit" value="0">
-        <input type="hidden" name="bug_id" value="{{ release.bug_list }}">
-        <p>If interested, please see the <button type="submit">complete list of changes</button> in this release.</p>
-      </form>
-      {% else %}
-          If interested, please see the  <a href="{{ release.get_bug_search_url() }}">complete list of changes</a> in this release.
-      {% endif %}
-        {{ release.text|markdown|safe }}
-      </p>
+    {% if release.version %}
+      <div class="latest-release" >
+        <div class="container">
+          {% block notes_heading_secondary %}
+          <div class="version">
+            <h3>{{ _('{version}')|f(version=release.version) }}</h3>
+            <h4>{{ release.product }} {{ release.channel }}</h4>
+            <p>{{ _('{date}')|f(date=release.release_date|l10n_format_date) }}</p>
+          </div>
+          <div class="description">
+              <h2>{{ _('Version {version}, first offered to {channel} channel users on {date}')|f(channel=release.channel, date=release.release_date|l10n_format_date, version=release.version) }}</h2>
+              <p>{{ release.text|markdown|safe }}</p>
+          </div>
+          {% endblock %}
+        </div>
+      </div>
+    {% endif %}
     </header>
   {% endblock %}
 
-    <div class="main-column">
-      {% if new_features %}
-      <section class="notes-section" id="new-features">
-        <h3>{{ _('What’s New') }}</h3>
-        <ul class="section-items tagged">
-          {% for note in new_features %}
-          <li {% if not note.tag %}class="untagged"{% endif %} id="note-{{ note.id }}">
-            {% if note.tag %}
-              <b class="tag tag-{{ note.tag.lower() }}">{{ note.tag }}</b>
+    <section class="notes-section">
+      <div class="features" id="new-features">
+        <div class="container">
+        {% if new_features %}
+          {% for note in new_features if note.tag == "New" %}
+            {% if loop.first %}
+            <div class="section-wrapper">
+              <p id="{{ note.tag }}">{{ note.tag }}</p>
```

Comment:
> I'd also also convert the ID to lowercase (I think it's just lower(note.tag)?) and update the corresponding CSS rules (since CSS selectors are case-sensitive).

Good idea. For the record, it was `note.tag|lower()` rather than `lower(note.tag)`.